dump-pack-ftp-unpack-restore/roles/unpack-restore-data-sets/templates/adrdssu-restore.j2

```diff
@@ -0,0 +1,17 @@
+{# Copyright (c) IBM Corporation 2020 #}
+ RESTORE -
+    INDD(SOURCE) -
+{% if output_volume is defined and output_volume %}
+    OUTDY({{output_volume | upper}}) -
+{% endif %}
+    DS(INCL( -
+{% for data_set in data_sets_to_restore %}
+        {{ data_set | upper }}{{ " - " if loop.index != loop.length else ")) -" }}
+{% endfor %}
+{% if replace is defined and replace %}
+    REPLACE -
+{% endif %}
+    CATALOG {{ "-" if new_hlq is defined and new_hlq else "" }}
+{% if new_hlq is defined and new_hlq %}
+    RENUNC({{ new_hlq | upper }})
+{% endif %}
```

Comment:
Missing new line here

Reply:
newline here may impact behavior of MVS program since it is used for input, will leave out if okay with you

Reply:
actually, it does not appear to have negative impact on other templates, will add 😄